WebRoot/css/zm.css

```diff
@@ -1975,14 +1975,48 @@ div.CompDragArea * {
 	display:inline-block;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+div.newTaskBanner div.ImgAdd-svg {
```

Comment:
As discussed, to maintain the naming convention and not to increase the specificity for this selector, removed div tag on classname